src/js/components/Data/stories/Simple.js

```diff
@@ -19,7 +19,20 @@ export const Simple = () => (
       status="info"
       message="Data is in 'beta'. The API surface is subject to change."
     />
-    <Data data={DATA} toolbar>
+    <Data
```

Comment:
https://user-images.githubusercontent.com/54560994/210440333-92af151d-0781-43ef-b1aa-de7671c640d3.mp4

After selecting a view and then searching, the filters from the view are still applied but the view is no longer shown as active in the Select. This makes it a little confusing what filters are applied and how to get back to where you started

Reply:
Good feedback. This should be addressed now.

Reply:
I'm no longer seeing this issue with the 'latest' view, but I am still seeing the issue on the 'behind' view

Reply:
Good catch. Should be fixed now.